config/locales/roles.new.pt_BR.yml

```diff
@@ -0,0 +1,44 @@
+# License: CC0-1.0
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
+#
+# To use the locales, use `I18n.t`:
+#
+#     I18n.t 'hello'
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
+# To learn more, please read the Rails Internationalization guide
+# available at https://guides.rubyonrails.org/i18n.html.
+
+pt_BR:
+  roles:
+    new:
+      invite_a_new_user: Convide um Novo Usuário
+      nonprofit_admin: Administrador da Organização
```

Comment:
The literal translation of "nonprofit" does not make sense in Brazilian Portuguese, so I translated to "Organização", which is equivalent to "Organization". What do you think?

Reply:
In some places they're called non-governmental organizations so is there a word for that? But you really know best so whatever you decide is good in my book 🙂

Reply:
One thing to consider is making sure whatever phrasing you use is consistent. It probably should be documented somewhere that this is the word to use.

Reply:
For non-governmental organizations it would be ONGs (initials for Organização Não Governamental), but it is technically wrong to call that - at least in Brazil - if they are not formally instituted like that, that's why I thought about "Organizations", which would be a more broad name.

> It probably should be documented somewhere that this is the word to use.

We could have a glossary for that.

Reply:
I think Organização is good then!